web-app/src/components/errors/ErrorBoundary.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import ErrorPage from "../../pages/errorPage";
+
+export default class ErrorBoundary extends React.Component {
```

Comment:
Does this need to be a class component? Would be nice if it was a functional component to keep things consistent

Reply:
https://reactjs.org/docs/error-boundaries.html#introducing-error-boundaries

"Only class components can be error boundaries."

There is a package which makes it possible to have a func component as the handler but its not worth imo (npm i react-error-boundary)

Reply:
Didn't know that! Looks good then :)